generator-jhipster-gomicro/generators/server/templates/gomicro/gomicro/eurekaregistry/helper/HttpHelper.go

```diff
@@ -46,13 +42,10 @@ func MakePostCall(urlToPost string, body OperationBody, headers map[string]strin
 		logger.Errorf("Error while making post call " + err.Error())
 		return errors.New("Error while making post call " + err.Error()), nil
 	}
-	logger.Infof("Successfull POST call with HTTP Status : " + resp.Status)
 	return nil, resp
 }
 
 func MakePutCall(urlToPost string, body OperationBody, headers map[string]string) (error, *http.Response) {
-	msg := fmt.Sprintf("In MakePostCall to %s with body %s", urlToPost, body)
```

Comment:
why remove a log for the PUT call for eureka ?
in spring boot application provided by the Jhipster we do have logs for it !